bazel/python_rules.bzl

```diff
@@ -163,7 +163,7 @@ def py_proto_library(
         _generate_py(
             name = codegen_grpc_target,
             deps = deps,
-            plugin = "//:grpc_python_plugin",
+            plugin = "@com_github_grpc_grpc//:grpc_python_plugin",
```

Comment:
(Minor) I believe the idiomatic way of doing it is using [Label](https://docs.bazel.build/versions/master/skylark/lib/Label.html#Label) constructor directly:
```python
plugin = Label("//:grpc_python_binary"),
```
This allows resolving the label target correctly regardless of the actual name used for importing the repo.

Reply:
Done. Thanks! I hadn't seen Label before.